src/Makefile.am

```diff
@@ -191,6 +191,8 @@ BITCOIN_CORE_H = \
   governance/validators.h \
   governance/vote.h \
   governance/votedb.h \
+  gsl/assert.h \
+  gls/pointers.h \
```

Comment:
```suggestion
  gsl/pointers.h \
```

Reply:
whoops :D I hate that issues like this still compiles locally